blocks/callout/callout.css

```diff
@@ -0,0 +1,109 @@
+.section .callout-wrapper {
+  padding: 0;
+  margin-bottom: 50px;
+  max-width: 1224px;
+  width: 100%;
+}
+
+.callout {
+  > div {
+    display: flex;
+    flex-direction: column;
+    background: var(--theme-clr-background);
+    height: 682px;
+
+    .callout-image {
+      flex: 1;
+      height: 322px;
+  
+      picture, picture img {
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+      }
+    }
+
+    .callout-content {
+      color: var(--theme-clr-text);
+      padding: 80px 15px;
+  
+      h3 {
+        font-family: var(--heading-font-family);
+        font-size: 22px;
```

Comment:
Using `font-size: 1.5rem` we could achieve similar font size as it's present on customer website.